compute_tools/src/compute.rs

```diff
@@ -50,6 +51,19 @@ pub struct ComputeNode {
     pub state: RwLock<ComputeState>,
 }
 
+#[derive(Debug, Serialize)]
+pub struct InsightsRow {
+    pub query: String,
+    pub total_runtime: String,
+    pub mean_runtime: String,
+}
+
+#[derive(Serialize)]
+pub struct Insights {
+    count: u64,
+    statements: Vec<InsightsRow>,
```

Comment:
I didn't see where you used these structs.

Reply:
Good catch, thanks. Some leftover from the previous version of the patch